nautobot/virtualization/migrations/0001_initial.py

```diff
@@ -383,7 +383,7 @@ class Migration(migrations.Migration):
                 "verbose_name": "interface",
                 "ordering": (
                     "virtual_machine",
-                    nautobot.utilities.query_functions.CollateAsChar("_name"),
+                    "_name",
```

Comment:
What is the reason for this change?

Reply:
I can attest that the reason for this is and other `_name` fields is eliminating `CollateasChar` since it's hard-coded to PG as a backend using the `COLLATE C` function. This doesn't exist in MySQL.